airbyte-webapp/src/hooks/services/useDocumentation.ts

```diff
@@ -22,6 +22,8 @@ const useDocumentation = (documentationUrl: string): UseDocumentationResult => {
       enabled: !!documentationUrl,
       refetchOnMount: false,
       refetchOnWindowFocus: false,
+      retry: false,
+      suspense: false,
```

Comment:
what is suspense mode?

Reply:
in terms of react-query: it is suspense mode from [React](https://reactjs.org/docs/concurrent-mode-suspense.html). It is enabled in the app by default but in this particular case we don't need it as we want to handle loading and error manual. 

Main difference is that in suspense mode, dom rendering will be postponed until the request is resolved ( nearest `<Suspense />` component will be displayed instead while something is loading. Without suspense mode - we could achieve the same by `loading, data, error` props.